src/ResourceGraph/ResourceGraph/help/Search-AzGraph.md

```diff
@@ -126,6 +138,20 @@ Type: System.Int32
 Parameter Sets: (All)
 Aliases:
 
+Required: False
```

Comment:
This portion of the file is automatically generated (other than the description), so the right way to update it would be to regenerate the help.

The example above and the description below are both portions that can be edited in the markdown file directly.

Reply:
My bad. I have removed that part now, and tried to regenerate the help but Include parameter is not added in build process.